lib/nostrum/struct/voice_ws_state.ex

```diff
@@ -16,4 +18,65 @@ defmodule Nostrum.Struct.VoiceWSState do
     :heartbeat_interval,
     :heartbeat_ref
   ]
+
+  @typedoc "The guild id"
```

Comment:
I think it would be nice if we could write this out like this:

```suggestion
  @typedoc "The guild id which this event applies to"
```

What do you think?